test/e2e/specs/specs-playwright/wp-signup__screenshot.ts

```diff
@@ -0,0 +1,284 @@
+/**
+ * @group legal
```

Comment:
FYI, this file should be in `test/e2e/specs/specs-legal` or somewhere else entirely, since it's not an end-to-end test.

This can be addressed in a follow-up PR.